modules/java_api/samples/face_detection_sample_async/Main.java

```diff
@@ -232,12 +236,13 @@ public void run() {
                     Imgproc.putText(img, label1, new Point(10, 80), 0, 0.7, color, 1);
                 }
                 HighGui.imshow("Detection", img);
+                HighGui.waitKey(1);
```

Comment:
Can we stop application by button? If no, can you try
```java
if (HighGui.waitKey(1) != -1) 
    break;
```
see https://docs.opencv.org/master/d7/dfc/group__highgui.html#ga5628525ad33f52eab17feebcfba38bd7

Reply:
It doesn't work for Java as it should. But it's ok for python...

Reply:
Well, which values are returned by `waitKey` ?